packages/core/core-flows/src/cart/workflows/list-shipping-options-for-cart.ts

```diff
@@ -39,6 +30,7 @@ export const listShippingOptionsForCartWorkflow = createWorkflow(
         "id",
         "sales_channel_id",
         "currency_code",
+        "customer.groups.id",
```

Comment:
question: If I am not mistaken, we have changed how customer group ID is passed in the pricing context – just want to make sure this is intentional.

We used to pass is as a top-level property:
```
{ ..., customer_group_id: "cusgrp_1234" }
```

Now, it seems we pass it via the cart:
```
{ cart: { customer: { groups: [ { id: "cusgrp_1234" } ] } } }
```

Is that correct, or have I missed something?

Reply:
This was leak from the draft work I had for the changes related to the customer groups i was talking about. Will revert this. 

All of our customer group rules are set as the attribute `customer_group_id`, which doesn't exist in the cart context. The second version you listed should be the actual rule path - `customer.groups.id`. This needs a data migration on price lists and prices, along with some changes in the pricing module, so thought will open a separate PR.